src/server/scripts/EasternKingdoms/Karazhan/karazhan.cpp

```diff
@@ -354,6 +354,7 @@ class npc_barnes : public CreatureScript
                     break;
                 case GOSSIP_ACTION_INFO_DEF + 2:
                     CloseGossipMenuFor(player);
+                    m_uiEventId = urand(1, 3);
```

Comment:
It must be random?, if so, maybe use `urand(EVENT_OZ, EVENT_RAJ)`

Reply:
Good point, better use enum if the numbers corresponds with existing enums (otherwise it looks like "magic" numbers).